HW8.py

```diff
@@ -0,0 +1,63 @@
+
+
+import sqlite3
+import csv
+import os
+import argparse
+
+DB_FILE = 'tasks_projects.db'
+
+
+def create_table_from_csv(csv_file_name, cur):
+#     print(csv_file_name)
+    table_name = csv_file_name.split('.')[0]  # Get table name from CSV file name
+#     print(table_name)
+    with open(csv_file_name, 'r') as f:
+        reader = csv.reader(f)
+#         print(reader)
+        headers = next(reader)  # first call of 'next' method reads first line containing columns names followed with data types, separated by colon (':')
+#         print(headers)
+        headers_with_types = ', '.join(headers).replace(':', ' ')  # make string like 'field1 varchar, field2 date' for insertion in CREATE statement
```

Comment:
It's not often practiced to keep metadata in the CSV header. But it's great that you realized that idea!